File: string-handling/string-position/file_extension.py
# Isaac Computer Science
# Usage licensed under the Open Government Licence v3.0

def extract_file_extention():
    """Demonstrates how to extract a file extension"""
    file_name = "monster.csv"
    dot_location = file_name.find(".")
    print(dot_location)
    position = dot_location + 1
    print(file_name[position:]) # Prints the remainder of the string
    

if __name__ == '__main__':
    extract_file_extention()

File: string-handling/string-position/file_extension.cs
/*
Isaac Computer Science
Usage licensed under the Open Government Licence v3.0

Note: This file is designed to be copied out and compiled on your machine. 
In order for it to compile properly you need to ensure that the project name is the same as the "namespace" in this file. 
To run this file you need to:
1. Copy the contents
2. Paste them into the C# IDE of your choice (Visual Studio, for example)
3. Compile the program
4. Run the program
*/

using System;


namespace StringHandling
{

    class StringPosition
    {

        // The Main method is the entry point for all C# programs
        public static void Main(string[] args)
        {
            string fileName = "monster.csv";
            string extension = GetExtension(fileName);
            Console.WriteLine(extension);
        }


        // Extracts everything after dot in file name
        public static string GetExtension(string fileName) {
            int position = fileName.IndexOf(".");
            int index = position + 1;
            string extension = fileName.Substring(index);
            return extension;
        }


    }
}
